### ISSUE TITLE
badge: Portuguese (Portugal) for SHUTDOWN

### ISSUE THREAD

**Issue by 0xShutdown** [revoltchat/translations#309]
### What language(s) did you translate?

Portuguese (Portugal)

### What is your Revolt ID?

01HWGQ4FCEYNE8RC9H7FBAGKTT

### Link to Weblate profile

https://translate.revolt.chat/user/SHUTDOWN/

### Validations

- [X] The language has a maintainer (see the full list [here](https://github.com/revoltchat/translations#languages)).
- [X] You agree that your translations are to a reasonable standard.

**Comment by DeclanChidlow** [comment]
If this request is still active and pending action, please let us know by replying to this comment. Thanks!

**Comment by 0xShutdown** [comment]
Yes, I am active. I've been busy, but I haven't started working on the translations yet.
